homeassistant/components/ddwrt/device_tracker.py

```diff
@@ -18,12 +19,21 @@
 
 DEFAULT_SSL = False
 DEFAULT_VERIFY_SSL = True
+DEFAULT_TYPE = 'lan'
```

Comment:
I think that it would make sense to it `wireless`. No braking change and most installations try to track the mobile users anyway.

Reply:
Sure thing, it was my original thought. Would it be merged faster without a breaking change?

Reply:
Reversed.